modules/product/src/ProductVariationManagerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\commerce_product\ProductVariationManagerInterface.
+ */
+
+namespace Drupal\commerce_product;
+use Drupal\commerce_product\Entity\Product;
+
+/**
+ * Defines the interface for product variation managers.
+ */
+interface ProductVariationManagerInterface {
+
+  /**
+   * Gets an array containing all the enabled variations of the given product.
+   *
+   * @param \Drupal\commerce_product\Entity\Product $product
```

Comment:
Needs description for the $product param (on the next line, indented, "The product.")